docs/sphinx/user_guide/feature/sort.rst

```diff
@@ -0,0 +1,141 @@
+
+   x = { a\ :sub:`0`\, b\ :sub:`0`\, a\ :sub:`1`\, ... }
+
+and calculates the output sequence:
+
+   y = { a\ :sub:`0`\, a\ :sub:`1`\, b\ :sub:`0`\, ... }
+
+An **unstable sort** may not preserve the order of equivalent elements and
+may produce either of the following output sequences.
+
+   y = { a\ :sub:`0`\, a\ :sub:`1`\, b\ :sub:`0`\, ... }
+
+   or
+
+   y = { a\ :sub:`1`\, a\ :sub:`0`\, b\ :sub:`0`\, ... }
+
+---------------------
+RAJA Unstable Sorts
+---------------------
+
+RAJA unstable sort operations look like the following:
+
+ * ``RAJA::sort< exec_policy >(container)``
+ * ``RAJA::sort< exec_policy >(container, comparator)``
+ * ``RAJA::sort< exec_policy >(iter, iter + N)``
+ * ``RAJA::sort< exec_policy >(iter, iter + N, comparator)``
+
+Here, 'container' is a range of elements and 'iter' is a random access
+iterator to a range of elements. 'container' and 'iter' provide access to the
+input sequence and contain the output sequence at the end of sort. The first
+and third sort operations above will be *non-decreasing* sorts since there is
+no comparator argument given; i.e., the sequences will be reordered *in-place*
+using operator::less. The second and fourth sorts will apply the comparator
+that is passed into the function.
+
+RAJA also provides sort pairs that operate on key, value pairs stored
+separately:
+
+ * ``RAJA::sort_pairs< exec_policy >(keys_container, vals_container)``
+ * ``RAJA::sort_pairs< exec_policy >(keys_container, vals_container, comparator)``
+ * ``RAJA::sort_pairs< exec_policy >(keys_iter, keys_iter + N, vals_iter)``
+ * ``RAJA::sort_pairs< exec_policy >(keys_iter, keys_iter + N, vals_iter, comparator)``
+
+Sort pairs generates the same output sequence of keys as sort and also reorders
+vals by permuting it the same as it did the keys; i.e. sorting the pairs based
+on their keys. The 'comparator' used in sort_pairs only compares keys.
+
+---------------------
+RAJA Stable Sorts
+---------------------
+
+Using RAJA stable sorts is essentially the same as unstable sorts:
+
+ * ``RAJA::stable_sort< exec_policy >(container)``
+ * ``RAJA::stable_sort< exec_policy >(container, comparator)``
+ * ``RAJA::stable_sort< exec_policy >(iter, iter + N)``
+ * ``RAJA::stable_sort< exec_policy >(iter, iter + N, comparator)``
+
+ * ``RAJA::stable_sort_pairs< exec_policy >(keys_container, vals_container)``
+ * ``RAJA::stable_sort_pairs< exec_policy >(keys_container, vals_container, comparator)``
+ * ``RAJA::stable_sort_pairs< exec_policy >(keys_iter, keys_iter + N, vals_iter)``
+ * ``RAJA::stable_sort_pairs< exec_policy >(keys_iter, keys_iter + N, vals_iter, comparator)``
+
+.. _sortops-label:
+
+--------------------
+RAJA Comparison Operators
+--------------------
+
+RAJA provides two operators that can be used to produce different ordered sorts:
+
+  * ``RAJA::operators::less<T>``
+  * ``RAJA::operators::greater<T>``
+
+.. note:: * All RAJA comparison operators are in the namespace
+            ``RAJA::operators``.
```

Comment:
This note looks a little weird in the generated docs. Better to omit the line break after 'namespace'.